app/includes/session.php

```diff
@@ -32,10 +32,11 @@ function admin_confirm_logged_in() {
 		<?php
 		}
 	}
-	
+
 	function message($msg="", $msgtype="") {
 	  if(!empty($msg)) {
-	    // then this is "set message"
+			$messgge="";
```

Comment:
I agree with your opinion, but the spelling is wrong

Reply:
I change it already